inc/Engine/Optimization/DelayJS/HTML.php

```diff
@@ -70,8 +70,8 @@ public function delay_js( $html ): string {
 		$this->excluded = array_map(
 			function( $value ) {
 				return str_replace(
-					[ '+', '?', '#' ],
-					[ '\+', '\?', '\#' ],
+					[ '+', '?ver', '#' ],
+					[ '\+', '\?ver', '\#' ],
```

Comment:
May I ask, why not using preg_quote method here?

Reply:
`preg_quote()` would be escaping characters we don't want to escape, as they are part of the patterns in the exclusion field.

Reply:
Those are the characters that can be part of an URL, and can cause a misinterpretation of the provided pattern when we check for the exclusion.

`?v` might happen, but it's a much rarer occurrence. Also, it can be manually removed from the pattern in the exclusion field if needed.